RenderStatePackager/src/RenderStatePackager.cpp

```diff
@@ -24,6 +24,9 @@
  *  of the possibility of such damages.
  */
 
+#include <deque>
+#include <sstream>
+
 #include "RenderStatePackager.hpp"
```

Comment:
Please have have the main header (`#include "RenderStatePackager.hpp"`) the first header in the file
https://github.com/DiligentGraphics/DiligentCore/blob/master/doc/CodingGuidelines.md#source-files